08-Secure by default/Secure Templates/ARM/Synapse.json

```diff
@@ -0,0 +1,113 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "synapseWorkspaceName":{
+            "type": "string",
+            "metadata": {
+                "description": "Synapse workspace name"
+            }
+        },
+        "dataLakeStorageAccountUrl":{
+            "type": "string",
+            "metadata":{
+                "description": "url of the data lake storage account which we need to associate with synapse workspace"
+            }
+        },
+        "dataLakeStorageFileSystem":{
+            "type": "string",
+            "metadata":{
+                "description": "File system name to associate with data lake storage and synapse workspace"
+            }
+        },
+        "sqlAdminUser":{
+            "type": "string",
+            "metadata":{
+                "description": "sql admin user "
+            }
+        },
+        "sqlAdminUserPassword":{
+            "type": "string",
+            "metadata":{
+                "description": "Password for sql admin user."
```

Comment:
NIT: 
1. Either remove or keep full stop (.) in every description.
2. Start description with uppercase  at everyplace.

Reply:
Do same in bicep template as well.

Reply:
Fixed.